examples/search_examples/public/search/app.tsx

```diff
@@ -109,6 +108,7 @@ export const SearchExamplesApp = ({
   const [currentAbortController, setAbortController] = useState<AbortController>();
   const [rawResponse, setRawResponse] = useState<Record<string, any>>({});
   const [selectedTab, setSelectedTab] = useState(0);
+  const [searchSourceWarnings, setSearchSourceWarnings] = useState<ResponseWarning[] | undefined>();
```

Comment:
why do we need this on state ?

Reply:
@ppisljar not sure what you suggest - we need either the warning or the inspector itself available in scope to the onclick handler that shows the warning on-demand.

Reply:
In the latest code, I have the warnings displayed in a new tab of the example app. They're populated into the content area of the tab as they show up in responses.